source/guides/clear/telemetrics.rst

```diff
@@ -272,7 +281,7 @@ Use :command:`-h` or :command:`--help` to view usage options.
    this, you can use an IP address or fully qualified domain name.
 
 
-Set up a back-end server to collect telemetry records
+Set up a backend server to collect telemetry records
 =====================================================
```

Comment:
one too many "=" since the word back-end was changed to backend

Reply:
Fixed